docs-md/apireference/c.md

```diff
@@ -4014,6 +4028,37 @@ mtev_hash_store(mtev_hash_table *h, const void *k, int klen, const void *data)
   mtev_hash_delete(), mtev_hash_delete_all() or mtev_hash_destroy().
  
 
+#### mtev_html_encode
+
+>Encode raw data as html encoded output into the provided buffer.
+
+```c
+int
+mtev_html_encode(const char *src, size_t src_len, char *dest, size_t dest_len)
```

Comment:
If I send mtev_html_encode/decode empty strings (size 0), the destination is successfully encoded to an empty string (size 0), but our return code is now an error (0) even though the operation was successful.  Do we care about this case and need to return `-1` on an error?

Reply:
trying to keep this consistent across encode/decode (like b32 and b64)